### PR TITLE
Fpv rcnn

### PR DESCRIPTION
**Pull request by DerrickXuNu**
Hey Yunshuang, I think it is ready to merge to our opencood main.

### CODE REVIEW

File: opencood/data_utils/datasets/__init__.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 from opencood.data_utils.datasets.late_fusion_dataset import LateFusionDataset
 from opencood.data_utils.datasets.early_fusion_dataset import EarlyFusionDataset
 from opencood.data_utils.datasets.intermediate_fusion_dataset import IntermediateFusionDataset
+from opencood.data_utils.datasets.intermediate_fusion_dataset_v2 import IntermediateFusionDatasetV2
```
Comment (YuanYunshuang) [inline opencood/data_utils/datasets/__init__.py:8]: So you did not merge IntermediateFusionDataset and V2 into one?

File: opencood/tools/train_utils.py [hunk 2]
```diff
@@ -70,8 +71,11 @@ def setup_train(hypes):
     folder_name = current_time.strftime("_%Y_%m_%d_%H_%M_%S")
     folder_name = model_name + folder_name
 
-    current_path = os.path.dirname(__file__)
-    current_path = os.path.join(current_path, '../logs')
+    if os.path.exists('/media/hdd/koko/experiments-output/opencood'):
```
Comment (YuanYunshuang) [inline opencood/tools/train_utils.py]: This can be removed.

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,17 @@
-matplotlib
-numpy
+cmake
+easydict~=1.9
+torch~=1.6.0
```
Comment (YuanYunshuang) [inline requirements.txt]: The versions can be removed. I also tested on torch 1.10, it seems to be working.

Comment (DerrickXuNu) [reply requirements.txt]: ok

File: opencood/data_utils/datasets/__init__.py (hunk 1 above)
Comment (DerrickXuNu) [reply opencood/data_utils/datasets/__init__.py:8]: yeah, i am thinking maybe keep both of them. How do you think

**Pull request merged**